ts/nni_manager/training_service/kubernetes/kubeflow/kubeflowConfig.ts

```diff
@@ -10,7 +10,7 @@ import { AzureStorage, KeyVaultConfig, KubernetesClusterConfig, KubernetesCluste
 } from '../kubernetesConfig';
 
 // operator types that kubeflow supported
-export type KubeflowOperator = 'tf-operator' | 'pytorch-operator' ;
+export type KubeflowOperator = string;  // 'tf-operator' | 'pytorch-operator'
```

Comment:
why we change this?

Reply:
The unpack logic before this yields string, and `tsc` complained a lot type mismatch, so...
Too lazy.